src/middleware/check_user_status.py

```
@@ -0,0 +1,28 @@
+                FROM check_user_status_rule
+                WHERE method = :method AND :path LIKE path
+            """
+            blacklist_rules = session.exec(text(raw_sql).bindparams(method=request.method, path=request.url.path)).all()  # type: ignore
+
+        if blacklist_rules:
+            try:
+                current_user = get_user(request)
+            except HTTPException as e:
+                return JSONResponse({"detail": e.detail}, e.status_code)
+            for rule in blacklist_rules:
+                if rule.user_status == current_user.status: return JSONResponse({"detail": f"cannot access path={request.url.path} with user status={current_user.status}"}, 403)
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)